### PR TITLE
Turn defvar to defcustom

### PR DESCRIPTION
**Pull request by wyuenho**
This PR allows the users to put customizations in their customs.el file.

### CODE REVIEW

File: magit-delta.el [hunk 1]
```diff
@@ -42,10 +53,14 @@ frame has a light or dark background. See `magit-delta-default-light-theme' and
...
-(defvar magit-delta-hide-plus-minus-markers t
-  "Whether to hide the +/- markers at the beginning of diff lines.")
+will be added if not present."
+  :type '(repeat string)
+  :group 'magit-delta)
+
+(defcustom magit-delta-hide-plus-minus-markers t
+  "Whether to hide the +/- markers at the beginning of diff lines."
+  :type '(choice (const :tag "Hide" t)
+                 (const :tag "Show" nil)))
```
Comment (dandavison) [inline magit-delta.el]: Is this one missing `:group 'magit-delta`?

**Review by dandavison — COMMENTED** [review]
Hi @wyuenho, nice to hear from you! They weren't very memorable, but you accepted a couple of small PRs from me in backgrid several years ago.

Thanks very much for this. I added one question inline, and just one final question: is it your understanding that this is 100% backwards compatible, in that users who are currently setting these variables in emacs lisp in their config file will not notice anything? (Empirically, my magit-delta config certainly seems to work just the same on this branch.)

**Comment by wyuenho** [comment]
Hi 👋 ! Long time!

> is it your understanding that this is 100% backwards compatible, in that users who are currently setting these variables in emacs lisp in their config file will not notice anything?

Yes, I submit drive-by defcustom PRs quite often lol

**Comment by dandavison** [comment]
Great, thanks a lot and best wishes for your current projects.

**Comment by wyuenho** [comment]
You too! Great work on Delta!

**Pull request merged**